rest_framework/fields.py

```diff
@@ -1098,17 +1127,8 @@ class ChoiceField(Field):
     }
 
     def __init__(self, choices, **kwargs):
-        # Allow either single or paired choices style:
-        # choices = [1, 2, 3]
-        # choices = [(1, 'First'), (2, 'Second'), (3, 'Third')]
-        pairs = [
-            isinstance(item, (list, tuple)) and len(item) == 2
-            for item in choices
-        ]
-        if all(pairs):
-            self.choices = OrderedDict([(key, display_value) for key, display_value in choices])
-        else:
-            self.choices = OrderedDict([(item, item) for item in choices])
+        flat_choices = [flatten_choice(c) for c in choices]
+        self.choices = OrderedDict(list(itertools.chain(*flat_choices)))
```

Comment:
Should we be leaving the `.choices` attribute alone, but adding a `.flatchoices` attribute?
What's the right way for us to access the grouping information, if we need to (eg for displaying a grouped select in the browsable API)